src/AudioUnit/AudioComponent.cs

```diff
@@ -342,6 +351,13 @@ public UIKit.UIImage GetIcon (float desiredPointSize)
 				return AudioComponentGetLastActiveTime (handle);
 			}
 		}
+
+		[Watch (7,0), TV (14,0), iOS (14,0)]
+		public UIImage CopyIcon () {
+			var ptr = AudioComponentCopyIcon (handle);
+			return ptr == IntPtr.Zero ? null : new UIImage (ptr);
```

Comment:
`AudioComponentCopyIcon` returns a retained handle, and it can also be an existing UIImage (for which we have a managed instance), so:

```suggestion
			return (UIImage) Runtime.GetNSObject (ptr, true);
```